install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -290,8 +294,10 @@ spec:
 {{- end }}
         - mountPath: /var/run/cilium
           name: cilium-run
+{{- if .Values.initResources }}
```

Comment:
This allows explicitly setting `initResources` to an empty dict to cause the resources to not be rendered.
I can also leave this change out if you prefer.

Reply:
> This allows explicitly setting `initResources` to an empty dict to cause the resources to not be rendered.
> I can also leave this change out if you prefer.

how could this happen exactly? It's not clear to me.

Reply:
A user could supply a values.yaml to the agent chart with following content:

```
initResources: {}
```